src/app/(proper_react)/redesign/(authenticated)/user/dashboard/View.tsx

```diff
@@ -115,6 +116,32 @@ export const View = (props: Props) => {
   const tabSpecificExposures = getTabSpecificExposures(selectedTab);
   const filteredExposures = filterExposures(tabSpecificExposures, filters);
 
+  // MNTOR-2226
+  const getExposureCardLink = (exposure: Exposure) => {
+    let link = "/redesign/user/dashboard";
```

Comment:
Q: is this default value used? I see some `if..else if..{if..else}..else` below, but I think every branch is overwriting this default value. Unless... there is a state where you're not eligible or a free scan (L125) _or_ eligible for premium (L128)... 🤔 

Conversely, do we need a temp var and should we just `return` early? (so instead of `link = 'abc'` just do `return 'abc'` and short circuit everything).